Makefile.build

```diff
@@ -0,0 +1,34 @@
+export GO111MODULE=on
+export GOPROXY=https://proxy.golang.org
+export CGO_ENABLED=0
```

Comment:
This was potentially new rather than a refactor, in case anything starts breaking.